test/test_helper.rb

```diff
@@ -292,9 +292,9 @@ def capture_transcript(gateway)
   end
 
   def dump_transcript_and_fail(gateway, amount, credit_card, params)
-    transcript = capture_transcript(gateway) do
+    transcript = gateway.scrub(capture_transcript(gateway) do
```

Comment:
🤔 I thought the purpose of this method was to dump the transcript in a raw form. The raw transcript is then used to assist in building the scrubbing logic for new gateways.

Reply:
Yes, meant to remove it. The scrub call was just some manual verification.